about.md

```diff
@@ -1,4 +1,6 @@
-Hello, I am Alex. My pronouns are they/them. After unfortunate recent events, I have picked up maintenance of this guide in order to continue the project AnonyPla started. Unfortunately, Lena passed during a complex surgery. We believe she went peacefully in her sleep and she did not suffer. Before her surgery, she gave me access to her accounts to maintain the guide. I will be maintaining her guide, as well as running her Matrix space, from this point. As I said before, I cannot guarantee I'll do everything right, or how she would have done it, but I'm trying my best.
+Hello, I am Alex. My pronouns are they/them. After unfortunate recent events, I have picked up maintenance of this guide in order to continue the project AnonyPla started. Unfortunately, Lena passed during a complex surgery. We believe she went peacefully in her sleep and she did not suffer. Before her surgery, she gave me access to her accounts to maintain the guide. I will be maintaining her guide, as well as administering her Matrix space. As I said before, I cannot guarantee I'll do everything right, or how she would have done it, but I'm trying my best.
+
+It's also no longer a solo project as it was before and I created the Anonymous Planet organization on GitHub. The guide is one of the projects of the Anonymous Planet organization as is the Privacy-Security-Anonymity matrix community. This organization is actively looking for potential collaborators. Feel free to propose your candidature if you feel you could help.
```

Comment:
"Propose your _candidature_" – try _candidacy_, or just "Feel free to apply", to sound less formal - more simplified.

From _The Economist_'s Style Guide:
_constituency_ – try _supporters_
_gubernatorial_ – try _governor’s_
_perception_ – try _belief_ or _view_
_rhetoric_ (of which there is too little, not too much) – try _language_
or _speeches_ or _exaggeration_ if that is what you mean.

```suggestion
It's also no longer a solo project, I created the Anonymous Planet organization on GitHub. The guide is one of the projects of the Anonymous Planet organization, as is the Privacy-Security-Anonymity Matrix community. This organization is actively looking for potential collaborators. Feel free to propose your candidacy if you want to help.
```